src/css/tui-editor-contents.css

```diff
@@ -237,3 +237,11 @@
     margin-right: 3.8px;
     margin-top: 3px;
 }
+
+.tui-editor-contents-placeholder:not(:focus):before {
+    content: attr(data-placeholder);
+    color:grey;
+    font-style: italic;
```

Comment:
Placeholder of markdown is not italic so I think this font style should be erased.